tests/std/tests/P2474R2_views_repeat_death/test.cpp

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#define _CONTAINER_DEBUG_LEVEL 1
+
+#include <ranges>
+
+#include <test_death.hpp>
+
+using namespace std;
+using ranges::repeat_view;
+
+void test_view_negative() {
+    [[maybe_unused]] auto v = repeat_view(1, -1); // bound must be positive
+}
+
+void test_iter_decrement() {
+    repeat_view v(1, 1);
+    auto it = v.begin();
+    --it; // can't decrement bound past 0
+}
+
+void test_iter_decrement_overflow() {
+    repeat_view v(1);
+    auto it = v.begin();
+    it += (numeric_limits<ptrdiff_t>::min)();
+    --it; // integer overflow
+}
+
+void test_iter_increment() {
+    repeat_view v(1);
+    auto it = v.begin();
+    it += (numeric_limits<ptrdiff_t>::max)();
+    it++; // integer overflow

```

Comment:
Nitpick: This is (somewhat inconsistently) testing postincrement; I think it should test preincrement which is where the actual check happens.